XenAdmin/Dialogs/VMProtectionRecovery/PolicyHistory.cs

```diff
@@ -169,7 +169,13 @@ private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
             if (_policy != null)
             {
                 if (comboBox1.SelectedIndex == 0)
-                    RefreshGrid(_policy.PolicyAlerts);
+		{

```

Comment:
I think we still need to call RefreshGrid(_policy.PolicyAlerts) for VMPP, because in this case the PolicyAlerts contains the most recent 10 alerts and there is no need for a xapi call (VMPP.get_alerts) in GetVMPPAlertsAction.